org.scintilla.SciTE.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: org.scintilla.SciTE
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: SciTE
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=host
+  
+modules:
+
+- name: SciTE
+  buildsystem: simple
+  sources:
+  - type: file
+    url: https://www.scintilla.org/scite446.tgz
+    sha256: 1c2e6eee6c8aa2e52983d9713a4b1c97d06c376324a3ec8932adfcb9e55d16d1
+    dest-filename: scite.tgz
+  build-commands:
+  - tar -zxf scite.tgz
+  - cd scintilla/gtk && 
+       make GTK3=1 -j $FLATPAK_BUILDER_N_JOBS prefix="${FLATPAK_DEST}"
+  - cd scite/gtk && 
+       make GTK3=1 -j $FLATPAK_BUILDER_N_JOBS prefix="${FLATPAK_DEST}" && 
+       make GTK3=1 install prefix="${FLATPAK_DEST}"
+  post-install:
+  - ln -sf ${FLATPAK_DEST}/lib/scite/*.so 
+           ${FLATPAK_DEST}/lib
+  
+  - cp ${FLATPAK_DEST}/share/applications/SciTE.desktop 
+       ${FLATPAK_DEST}/share/applications/org.scintilla.SciTE.desktop
+  
+  - sed 's/Sci48M/org.scintilla.SciTE/g' -i ${FLATPAK_DEST}/share/applications/org.scintilla.SciTE.desktop
```

Comment:
it's better to use `rename-desktop-file` and `rename-icon`.

and you can use `copy-icon: true` if you need to keep it where it is.

Reply:
as we are supplying icon, we don't need to rename. rename-desktop-file done